readme.md

```diff
@@ -16,114 +15,98 @@ Member States (from the European Economic Area, which encompasses more countries
+The sister organization [EU Digital Green Certificates](https://github.com/eu-digital-green-certificates) - managed by TSi - is the home of the official reference implementations of the DCC gateway, Holder and Verifier apps and associated tooling.
 
 A number of Member States have open sourced their implementations, including their signing services, Holder apps and Verifier apps. You can find an overview of them here.
 
-# Specifications
-
-[LINK to app spec repos]
-
-https://github.com/ehn-dcc-development/ehn-dcc-valuesets
-https://github.com/ehn-dcc-development/ehn-dcc-schema
-https://github.com/ehn-dcc-development/hcert-spec
-
-## Documentation
-
-https://github.com/ehn-dcc-development/hcert-trust [RYAN: deprecated]
-
-## Implementations
-
-[TODO]
-
-## Full Stack
-
-These repositories contain implementations of the entire DCC stack, and are the perfect starting place for anyone who wants to understand how the DCC system works in detail.
-They can also be used during developement to validate your own implementation.
-
-The full stack consists of:
-
-1. Creating the JSON payload conform the schema.
-2. Packing according the HCERT specifications (so sign + pack in COSE/CBOR, zip, base45 then format to HCERT string and optionally rendering as a QR).
-3. Validation by scanning QR (or processing HCERT), checking the signature and optionally checking the JSON payload against the schema.
-
-https://github.com/ehn-dcc-development/ehn-sign-verify-python-trivial
-
-
-## HCERT
-
-[Explain that HCERT implementation contains the entire stack]
-
-[TODO half of these repos contain an almost full stack impelementain]
-
-https://github.com/ehn-dcc-development/hcert-kotlin
-https://github.com/ehn-dcc-development/hcert-app-swift
-https://github.com/ehn-dcc-development/python-hcert
-https://github.com/ehn-dcc-development/hcert-service-kotlin
-https://github.com/ehn-dcc-development/hcert-dotnet
-https://github.com/ehn-dcc-development/hcert-java
-
-## COSE
-
-[Explain]
-
-https://github.com/ehn-dcc-development/cose-js
-https://github.com/ehn-dcc-development/COSE-JAVA
-
-
-## Base45
+## Specifications
 
-[Explain]
+The DCC specifications have been split into four repositiros:
 
-https://github.com/ehn-dcc-development/base45-php
-https://github.com/ehn-dcc-development/base45-cs
-https://github.com/ehn-dcc-development/base45-ansi-C
-https://github.com/ehn-dcc-development/base45-swift
-https://github.com/ehn-dcc-development/base45-js
-https://github.com/ehn-dcc-development/base45-java
+* [hcert-spec](https://github.com/ehn-dcc-development/hcert-spec) specifies the container and encoding formats used by the DCC.
+* [ehn-dcc-valuesets](https://github.com/ehn-dcc-development/ehn-dcc-valuesets) provides the valueset datastructures and a snapshot of the valuesets (the current versions are published on the DCCG Gateway.
```

Comment:
Maybe mention schema repo first, and then value sets? Would be the logical as well as dependency order.